### PR TITLE
INTYGFV-14625: Replace fontawesome with inera icons

### PR DESCRIPTION
**Pull request by SamuelTilly**

### CODE REVIEW

File: packages/webcert/src/feature/certificate/Buttons/ForwardCertificateButton.tsx [hunk 1]
```diff
@@ -60,7 +58,7 @@ const ForwardCertificateButton: React.FC<Props> = ({
         onClick={handleEmailSend}
         tooltip={description}
         data-testid="forward-certificate-button"
-        startIcon={<FontAwesomeIcon size="lg" icon={faShare} />}>
```
Comment (SopraErik) [inline packages/webcert/src/feature/certificate/Buttons/ForwardCertificateButton.tsx:63]: The forward arrow points in the wrong direction
Maybe use transform: scaleX(-1)  to reverse it
Also, the vertical alignment looks a bit off

image

**Review by SopraErik — CHANGES_REQUESTED** [review]

File: packages/webcert/src/feature/certificate/Buttons/ForwardCertificateButton.tsx (hunk 1 above)
Comment (SamuelTilly) [reply packages/webcert/src/feature/certificate/Buttons/ForwardCertificateButton.tsx:63]: Good catch, fixed this. I think the alignment looks a bit better when flipped
image

**Review by SopraErik — APPROVED** [review]
Nice job!

**Comment by SamuelTilly** [comment]
Something strange happening with eslint for this branch

**Pull request merged**